src/commands/nfts/burnNft.ts

```diff
@@ -50,13 +50,14 @@ export const burnNft = async (
       'nft721-sales'
     )
 
-    const nft = loadNftContract(config, nftAddress)
+    const nft = loadNftContract(config, nftAddress).connect(config.signer)
```

Comment:
there is no need for a custom function for this. you can use `nvm.contracts.loadNft721(address)`